js/gated-content/src/store/modules/personalTraining.js

```diff
@@ -46,14 +50,18 @@ export default {
     setRemoteLink(state, value) {
       state.remoteLink = value;
     },
+    setLocalName(state, value) {
+      state.localName = value;
+    },
   },
   getters: {
     isJoinedVideoSession: (state) => state.videoSessionStatus,
     isMeetingComplete: (state) => dayjs().isAfter(state.personalTrainingDate),
     personalTrainingId: (state) => state.personalTrainingId,
     isInstructorRole: (state) => state.instructorRole,
     remoteLink: (state) => state.remoteLink,
-    localName: (state) => (state.instructorRole
+    localName: (state) => state.localName,
```

Comment:
@hamrant Hi.
I do see saving in local settings, but cant see code that will patch user entity and write user name to Drupal

Reply:
@anpolimus there no such requirements in the task, but we can add this

Reply:
@anpolimus fixed.

> Accept only JSON:API read operations.

Because of this setting I implemented custom routes for user name update